etc/helm/pachyderm/templates/pachd/auth-secret.yaml

```diff
@@ -27,6 +26,9 @@ stringData:
     - profile
     - groups
     - openid
+{{ if not .Values.pachd.oauthClientSecretSecretName }}
+  pachd-oauth-client-secret: {{ default (randAlphaNum 32) .Values.pachd.oauthClientSecret }}
```

Comment:
In the past I've had bad luck putting newlines in places like this.  One of the helm tests ends up failing; if it passes you're good to go.